WELCOME.md

```diff
@@ -0,0 +1,47 @@
+👋 Hello @new_user_name and welcome to the For the Horde community! We're so happy you're here! The purpose of this community is to give you a space to connect with other Minnesota-based tech and tech-adjacent folks. Want to be successful here? Try to give more than you take.
+
+✅ First things first. This is a community and we want this to be a safe and inclusive space for everyone. We expect you to abide by the For the Horde Code of Conduct in this Slack team AND while participating in any in-person or online For the Horde events. Please head on over to https://git.io/v0d6b to check it out.
+
+Need to report Code of Conduct violation? The admins are listed at the bottom of the Code of Conduct. Snap a screenshot, report it.
+
+🚴🏿‍♀️🕺🏻👩‍💻😻⚾️✈️📺 Second, you've automagically been added to the #introductions channel. Please introduce yourself to the community there and share something fun about yourself. Where you're from, what you're working on, what your most peculiar hobby is, what your favorite color is, etc... We want to get to know you! Also, there's probably a channel for that peculiar hobby.
+
+📣 Speaking of channels. We have a lot. When thinking about where to post a message, go through the channel list and check there first. It’s very tempting with ~1200 eyeballs to post something to #general. We tend to treat #general like a watercooler, not a place to garner attention.
+
+_Popular channels:_
+
+- #frontend - talk about your javascript
+- #devs-n-ops - do not talk about your javascript
```

Comment:
I think we can re-word the #frontend and #dev-n-ops descriptions so they read as more _inclusive_ and less snarky (no offense Jake).

Here's my stab at it:
```
- #devs-n-ops - a safe place to talk about development & ops related stuff (read: not front-end javascript)
- #frontend - a safe place to talk about your javascript :simple_smile:
```

Reply:
I hesitated at the #devs-n-ops description too, but didn't suggest a change because fun and personality in this message is valuable. I'm still not opposed to updating the #devs-n-ops description, but think the light/joke-y tone is a feature and not a bug.

Reply:
I'm going to noodle on this more. I don't think `:face_with_rolling_eyes:` fixes this, but I also understand where you're coming from. One thing I also considered (because I wanted to ask in one of these channels about it the other day and didn't know where to put it) is that javascript IS a backend technology.